gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-plans.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { ADMIN_USER, API_PUBLISHER_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { PlansFaker } from '@gravitee/fixtures/management/PlansFaker';
+import { ApiEntity, PlanStatus } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+
+import homePage from "@pageobjects/HomePage/HomePage"
+import apiHome from "@pageobjects/Apis/ApiHome"
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+
+// This is purely for PageObject example and on hold for now until policy studio changes are completed.
+
+
+    describe('API Plans Feature', () => {
+        beforeEach(() => {
+          cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+        });
+
+        const HomePage = new homePage()
+        const ApiHome = new apiHome()
+        const ApiDetails = new apiDetails()
+
+        it('Verify Home Page', () => {
+            cy.visit('/')
+            cy.wait(1000)
+            cy.contains('Home board').should('be.visible');
+        });
+        
+        it('Verify Existing Plan', () => {
+            cy.visit('/#!/environments/default/apis/?q=Test&page=1&size=10');
+            cy.getByDataTestId('search').click().type("Test");
+            cy.get('input').type('{enter}')
```

Comment:
Not sure what's happening here. Pressing enter on a button?

Reply:
Pretty sure this is how I was negating the double click when navigating to API, because cypress for some reason won't let me just click the API in list before finishing in that text box...?